osf/features.py

```diff
@@ -1,49 +1,10 @@
-flags = {
-    'STORAGE_USAGE': 'storage_usage',
-    'INSTITUTIONAL_LANDING_FLAG': 'institutions_nav_bar',
-    'STORAGE_I18N': 'storage_i18n',
-    'OSF_GROUPS': 'osf_groups',
-    'ENABLE_CHRONOS': 'enable_chronos',
-    'EGAP_ADMINS': 'egap_admins',
-    'EMBER_AB_TESTING_HOME_PAGE_VERSION_B': 'ab_testing_home_page_version_b',
-    'EMBER_AB_TESTING_HOME_PAGE_HERO_TEXT_VERSION_B': 'ab_testing_home_page_hero_text_version_b',
-    'EMBER_AUTH_REGISTER': 'ember_auth_register',
-    'EMBER_CREATE_DRAFT_REGISTRATION': 'ember_create_draft_registration_page',
-    'EMBER_EDIT_DRAFT_REGISTRATION': 'ember_edit_draft_registration_page',
-    'EMBER_FILE_REGISTRATION_DETAIL': 'ember_file_registration_detail_page',
-    'EMBER_FILE_PROJECT_DETAIL': 'ember_file_project_detail_page',
-    'EMBER_MEETINGS': 'ember_meetings_page',
-    'EMBER_MEETING_DETAIL': 'ember_meeting_detail_page',
-    'EMBER_MY_PROJECTS': 'ember_my_projects_page',
-    'EMBER_PROJECT_ANALYTICS': 'ember_project_analytics_page',
-    'EMBER_PROJECT_CONTRIBUTORS': 'ember_project_contributors_page',
-    'EMBER_PROJECT_DETAIL': 'ember_project_detail_page',
-    'EMBER_REGISTRATION_FILES': 'ember_registration_files_page',
-    'EMBER_PROJECT_FILES': 'ember_project_files_page',
-    'EMBER_PROJECT_FORKS': 'ember_project_forks_page',
-    'EMBER_PROJECT_REGISTRATIONS': 'ember_project_registrations_page',
-    'EMBER_PROJECT_SETTINGS': 'ember_project_settings_page',
-    'EMBER_PROJECT_WIKI': 'ember_project_wiki_page',
-    'EMBER_REGISTRATION_FORM_DETAIL': 'ember_registration_form_detail_page',
-    'EMBER_REGISTRIES_DETAIL_PAGE': 'ember_registries_detail_page',
-    'EMBER_SEARCH_PAGE': 'ember_search_page',
-    'EMBER_USER_PROFILE': 'ember_user_profile_page',
-    'EMBER_USER_SETTINGS': 'ember_user_settings_page',
-    'EMBER_USER_SETTINGS_ACCOUNTS': 'ember_user_settings_account_page',
-    'EMBER_USER_SETTINGS_ADDONS': 'ember_user_settings_addons_page',
-    'EMBER_USER_SETTINGS_APPS': 'ember_user_settings_apps_page',
-    'EMBER_USER_SETTINGS_NOTIFICATIONS': 'ember_user_settings_notifications_page',
-    'EMBER_USER_SETTINGS_TOKENS': 'ember_user_settings_tokens_page',
-}
+import yaml
 
-switches = {
-    'ENABLE_INACTIVE_SCHEMAS': 'enable_inactive_schemas',
-    'OSF_PREREGISTRATION': 'osf_preregistration',
-    'DISABLE_ENGAGEMENT_EMAILS': 'disable_engagement_emails',
-    'ELASTICSEARCH_METRICS': 'elasticsearch_metrics',
-    'ENFORCE_CSRF': 'enforce_csrf',
-    'ENABLE_RAW_METRICS': 'enable_raw_metrics',
-}
+with open('osf/features.yaml', 'r') as stream:
```

Comment:
nit: Make the file-path a Module-level constant, something like
`WAFFLE_VALUES_YAML = 'osf/features.yaml`

Then this can be imported in the management command and elsewhere instead of needing to know the path.